services/gundeck/test/unit/Native.hs

```diff
@@ -63,7 +65,8 @@ serialiseOkProp t = ioProperty $ do
             plain <- EVP.cipherBS c (encKeyBytes encKey) iv EVP.Decrypt dat'
             let n' = plainNotif <$> decodeStrict' plain
             return $ n' == Just n && mac == cipherMac p
-    return $ equalTransport && equalNotif
+    let debugInfo = (t, a, n, {- c, d, m, -} r, sn, equalTransport, equalNotif)
```

Comment:
Consider adding variable names, e.g.

```haskell
let debugInfo = intercalate ", " $ zipWith (\name var -> name ++ "=" ++ var)
      [ ("something something", t)
      , ("dunno what", a)
      , ... ]
return $ counterexample debugInfo (equalTransport && equalNotif)
```

Reply:
i'd rather not spend the time.  this only happens if a test case fails, in which case we need to take a closer look at the source anyway.